fury/actor.py

```diff
@@ -444,6 +444,63 @@ def contour_from_roi(data, affine=None,
     return skin_actor
 
 
+def contour_from_label(data, affine=None,
+                       color=None, opacity=None):
```

Comment:
Can you remove `opacity`? it is not needed because `color` should be RGB or RGBA (manage the opacity)

Reply:
Hello @skoudoro , so basically we want to take 3 input arguments and calculate the opacity value internally via the color array?

Reply:
I am actually a bit confused with what to do with the opacity parameter.

Reply:
I have implemented a solution hopefully it fulfils the requirement.